### DISCUSSION TITLE
home assistant configuration

### DISCUSSION THREAD

**Discussion by twostroker84** [philibertc/micronova_controller#24] [Q&A] [answered]
Hi everyone 
i have some problem to configure my home assistant.
I copy the two string in my configuration.yaml i change the name where there is the comment, but i can't understand what i have to edit in the line 4 and 5.
Thanks

**Comment by philibertc** [comment, marked as answer]
Hello, you must edit all the lines "state_topic:" and "command_topic:" so that "mqtt_topic" corresponds to the topic you have chosen during [the configuration](https://github.com/philibertc/micronova_controller#configure-wifi-and-the-mqtt-broker-url). The default value for "mqtt_topic" is "micronova".
Sincerely,
philibertc

**Reply by testeur70** [reply]
hello, I am also trying to use home assiatnt since yesterday night and have the same question. 
Does it mean that if we sent the micronova_controller.ino file on github without any change those two lines will be in configuration.yaml:
`    state_topic: "micronova/onoff"    #You have to edit that
    command_topic: "micronova/intopic"    #Also that
`
Regards
Testeur


Edit : ok I have my answer, I changed my configuration.yaml file as described in this topic and can now swithch on my stove from Home Assistant. 

Edit 2 : @Philibertc This project is now compatible with PIAZZETTA stoves. I have the P958D model witha micronova MB.

**Reply by philibertc** [reply]
Hello, thanks for the feedback!
